vehicle/GasEngine.h

```diff
@@ -5,18 +5,18 @@
 
 class GasEngine final : public IEngine {
 public:
-    GasEngine(int cylinders) : cylinders(cylinders) {}
+    GasEngine(int m_cylinders) : m_cylinders(m_cylinders) {}
```

Comment:
same here

Reply:
updated, p_ for parameters, m_ for class members